ksql-engine/src/main/java/io/confluent/ksql/util/PersistentQueryMetadata.java

```diff
@@ -30,9 +31,9 @@
   public PersistentQueryMetadata(String statementString, KafkaStreams kafkaStreams,
```

Comment:
`final` params and new line for each param when it is over multiple lines

Reply:
Done!